package.json

```
@@ -38,7 +38,7 @@
     "build-js-client": "tsc --declaration false --outDir js-dist"
   },
   "dependencies": {
-    "@grpc/grpc-js": "~1.12.5",
+    "@grpc/grpc-js": "^1.12.5",
```

```suggestion
    "@grpc/grpc-js": "^1.13.0",
```
If we don't move the pin forward, there's a potential that a consumer's lockfile will install a 1.12.x package that's compatible with this pin and then they'll have a weird failure.